osl/interface_linux.go

```diff
@@ -290,7 +291,17 @@ func (n *networkNamespace) AddInterface(srcName, dstPrefix string, options ...If
 	}
 
 	// Up the interface.
-	if err := nlh.LinkSetUp(iface); err != nil {
+	maxSetupRetry := 3
+	retryCount := 0
+	for ; retryCount < maxSetupRetry; retryCount++ {
+		if err := nlh.LinkSetUp(iface); err == nil {
+			break
+		} else {
+			log.Debugf("error in setup link up, error %+v", err)
+		}
+		time.Sleep(1 * time.Second)
```

Comment:
yeah, tens of milliseconds works for me too, I commit it agent as your suggestion.